### PR TITLE
Remove previous MDS root certificate if refreshed

### PR DESCRIPTION
**Pull request by ChaitanyaKulkarni28**

### CODE REVIEW

File: google_guest_agent/agentcrypto/mtls_mds_windows.go [hunk 1]
```diff
@@ -59,6 +59,11 @@ var (
 
 // writeRootCACert writes Root CA cert from UEFI variable to output file.
 func (j *CredsJob) writeRootCACert(_ context.Context, cacert []byte, outputFile string) error {
+	num, err := serialNumber(outputFile)
```
Comment (dorileo) [inline google_guest_agent/agentcrypto/mtls_mds_windows.go:62]: Can we move this closer to line 92? It only makes sense there, the logging can also be done within that if statement.

**Comment by dorileo** [comment]
Just nit picking comment, otherwise looking good.

File: google_guest_agent/agentcrypto/mtls_mds_windows.go (hunk 1 above)
Comment (ChaitanyaKulkarni28) [reply google_guest_agent/agentcrypto/mtls_mds_windows.go:62]: if we receive new creds, this file will be overwritten by that time. So we need to make sure previous one is read before we attempt to write

Comment (dorileo) [reply google_guest_agent/agentcrypto/mtls_mds_windows.go:62]: Can you add a comment explaining that?

Comment (ChaitanyaKulkarni28) [reply google_guest_agent/agentcrypto/mtls_mds_windows.go:63]: Done

**Pull request merged**